tools/benchmark/CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 ### ⚠ BREAKING CHANGES
 
 Update `typescript` dependency from `4.x` to `5.x`.
+Update `version` from `0.50.0` to `0.51.0`.
```

Comment:
We need a new section at the top for 0.51.0 instead of adding to the details of what happened in 0.50.

I'd put this in the new section:

```
### ⚠ BREAKING CHANGES

Mocha reporters have been consolidated into a single one that can handle arbitrary properties through `BenchmarkData.customData`, plus `BenchmarkData.customDataFormatters` to specify how each value should be printed to console.
Consumers who previously used `MochaMemoryTestReporter.js` should now use `MochaReporter.js`.
```

Reply:
Updated

Reply:
Hmm but you replaced the section for 0.50.0 and what happened then. The new content should go in a new section :)